workers/unity/Packages/com.improbable.gdk.core/Collections/BlittableString.cs

```diff
@@ -0,0 +1,24 @@
+namespace Improbable.Gdk.Core.Collections
+{
+    public struct BlittableString
+    {
+        private readonly uint handle;
+
+        internal string Internal => StringProvider.Get(handle);
+
+        internal BlittableString(uint handle)
+        {
+            this.handle = handle;
+        }
+
+        public void Set(string value)
```

Comment:
Unless this is the only thing you need you'll always have to assign it to a local string variable to use it (except for passing it to functions)